### PR TITLE
chore: update dependencies

### PR DESCRIPTION
**Pull request by GChuf**
Fixes 10 low vulnerabilities , updates dependencies.

### CODE REVIEW

**Comment by remy** [comment]
Does this fix production vulns - because I didn't think nodemon was carrying any.

If I'm doing local dev, then I think some of the packages are marked as potentially vulnerable but that seems to only really affect me…

**Comment by GChuf** [comment]
Yeah, that's only for local dev I guess. But hopefully the performance of nodemon increases as well, I haven't tested that ...

File: package.json [hunk 1]
```diff
@@ -42,10 +42,10 @@
     "postinstall": "node bin/postinstall || exit 0"
   },
   "devDependencies": {
-    "@commitlint/cli": "^3.1.3",
+    "@commitlint/cli": "^3.2.0",
     "@commitlint/config-angular": "^3.1.1",
-    "async": "1.4.2",
-    "coffee-script": "~1.7.1",
+    "async": "^1.5.2",
+    "coffee-script": "~1.12.7",
```
Comment (GeoffreyBooth) [inline package.json:48]: Please change this to `"coffeescript"`, the project migrated.

**Comment by GChuf** [comment]
oops - updated just the package.json file
will fix it so that package-lock gets updated as well later today

**Comment by GChuf** [comment]
OK - squashed commits and updated coffee-script to coffeescript. Tests passed.

**Pull request closed without merging**